File: BuilderDesignPattern/Desktop_Example/HpDesktopBuilder.java
package BuilderDesignPattern.Desktop_Example;

public class HpDesktopBuilder extends DesktopBuilder{

	public void buildMonitor() {
		desktop.setMonitor("HP MONITOR");
	}

	public void buildKeyboard() {
		desktop.setKeyboard("HP KEYBOARD");
	}

	public void buildMouse() {
		desktop.setMouse("HP MOUSE");
	}

	public void buildSpeakers() {
		desktop.setSpeakers("HP SPEAKERS");
	}

	public void buildRam() {
		desktop.setRam("HP RAM");
	}

	public void buildProcessor() {
		desktop.setProcessor("HP PROCESSOR");
	}

	public void buildMotherboard() {
		desktop.setMotherboard("HP MOTHERBOARD");
	}
}

File: BuilderDesignPattern/Destop_Example_Python/Builders/HPDesktopBuilder.py
from BuilderDesignPattern.Destop_Example_Python.Builders.DesktopBuilder import DesktopBuilder


class HPDesktopBuilder(DesktopBuilder):

	def buildMonitor(self) -> None:
		self._desktop.setMonitor("HP Monitor")

	def buildKeyboard(self) -> None:
		self._desktop.setKeyboard("HP Keyboard")

	def buildMouse(self) -> None:
		self._desktop.setMouse("HP Mouse")

	def buildSpeakers(self) -> None:
		self._desktop.setSpeakers("HP Speakers")

	def buildRam(self) -> None:
		self._desktop.setRam("HP Ram")

	def buildProcessor(self) -> None:
		self._desktop.setProcessor("HP Processor")

	def buildMotherboard(self) -> None:
		self._desktop.setMotherboard("HP Motherboard")